src/modules/site-v2/base/views/tools/genetic_mapping.py

```diff
@@ -22,45 +30,79 @@ def genetic_mapping():
   """
       This is the mapping submission page.
   """
-  title = 'Genetic Mapping'
-  alt_parent_breadcrumb = {"title": "Tools", "url": url_for('tools.tools')}
-  jwt_csrf_token = (get_jwt() or {}).get("csrf")
-  form = FileUploadForm()
-  # TODO: change
-  nemascan_container_url = 'https://github.com/AndersenLab/dockerfile/tree/nemarun/nemarun'
-  nemascan_github_url = 'https://github.com/AndersenLab/NemaScan'
-  return render_template('tools/genetic_mapping/mapping.html', **locals())
+  return render_template('tools/genetic_mapping/mapping.html', **{
+
+    # Page info
+    'title': 'Genetic Mapping',
+    'alt_parent_breadcrumb': {"title": "Tools", "url": url_for('tools.tools')},
+
+    # Form info
+    'jwt_csrf_token': (get_jwt() or {}).get("csrf"),
+    'form': FileUploadForm(),
+
+    # TODO: change
+    'nemascan_container_url': 'https://github.com/AndersenLab/dockerfile/tree/nemarun/nemarun',

```

Comment:
@vlagrassa we track the container info in DataStore. It might be helpful to add the 'git repo' as a new property to the container image and reference it here.